### PR TITLE
[RELEASE]: v0.9.4

### PR DESCRIPTION
**Pull request by vincentschen**
## Description of proposed changes
Update CHANGELOG.md + snorkel `version.py` to 0.9.4.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,14 +4,21 @@ All notable changes to this project will be documented in this file.
...
-### [Deprecated]
-### [Removed]
 
+* PR #1540: Fix squeeze bug in `to_int_label_array` function
+* PR #1535: Refactor baseline model imports in snorkel.labeling.model
+* PR #1520: Fix error bucket documentation in snorkel.analysis.error_analysis
+
+### [Contributors]
+
+Thanks to @ptrcklv for recent contributions!
```
Comment (henryre) [inline CHANGELOG.md:21]: #1523 was by @rjurney! (I just did some branch cleaning.

Comment (vincentschen) [reply CHANGELOG.md]: 🚀 awesome!! updated

**Comment by lukehsiao** [comment]
Will #1565 make it into this release, by chance?

**Comment by vincentschen** [comment]
> Will #1565 make it into this release, by chance?

#1565 will make it into 0.9.5, which we'll cut shortly after 0.9.4!

**Comment by lukehsiao** [comment]
> #1565 will make it into 0.9.5, which we'll cut shortly after 0.9.4!

Amazing. Thank you, Vincent! We will be able to release new version of Fonduer once 0.9.5 is out. This dependency upgrade is going to be super helpful.

**Pull request merged**